load-client/src/main/scala/client/TransactionGeneratorLoader.scala

```diff
@@ -0,0 +1,60 @@
+package client
+
+import scala.concurrent.duration._
+import akka.Done
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.{HttpEntity, HttpMethods, HttpRequest, HttpResponse, MediaTypes}
+import akka.stream.scaladsl.{Sink, Source}
+
+import scala.concurrent.Future
+
+object TransactionGeneratorLoader extends App {
```

Comment:
This is for sending requests to the https://github.com/mckeeh3/transaction-generator, which will generate messages to the Kafka topic.